packages/website/docs/04-Advanced-Guides/authentication.md

```diff
@@ -31,19 +36,26 @@ The response body of `GET /authentication`:
 ```
 
 ## Authentication Types
+
 There are four types of authentication: `email`, `oauth`, `oauthcallback`, and `signout`. Each authentication has a schema of this:
 
 ```json
 {
   "type": "email" | "oauth" | "oauthcallback" | "signout";
   "provider": string;
   "operatioId": string; // Used to determine how to send a request.
+  "mode"?: 'navigate' | 'cors'; // Use to determine how to open the Oauth endpoint. Only if type is oauth.
   "defaultParametersValue"?: any;
   "defaultRequestBodyValue"?: any;
 }
 ```
 
+:::tip　
+Basically, you should set `mode: 'cors'`.
+However, if you are self-hosting viron, you can choose any mode.
```

Comment:
Typically, you should set `mode: 'cors'`.
However, if you are self-hosting Viron, you have the flexibility to choose any mode.